<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meteorShowerDialog</class>
 <widget class="QWidget" name="meteorShowerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meteor Showers Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TitleBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>530</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="stelWindowTitle">
        <property name="text">
         <string>Meteor Showers Plug-in Configuration</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeStelWindow">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="updatesGroup">
         <property name="title">
          <string>Update catalog from Internet</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QCheckBox" name="internetUpdatesCheckbox">
            <property name="text">
             <string>Update from Internet sources</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="lastUpdateLabel">
              <property name="text">
               <string>Last update:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QDateTimeEdit" name="lastUpdateDateTimeEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="updateFrequencyLabel">
              <property name="text">
               <string>Update frequency (hours):</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QSpinBox" name="updateFrequencySpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="nextUpdateLabel">
              <property name="text">
               <string>[next update info]</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="updateButton">
              <property name="text">
               <string>Update now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="displayBox">
         <property name="title">
          <string>Settings for meteor showers</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="displayModeCheckBox">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Enable display of radiants of meteor showers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displayAtStartupCheckBox">
            <property name="text">
             <string>Enable display at startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displayShowMeteorShowerButton">
            <property name="text">
             <string>Show meteor showers button on toolbar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="restoreDefaultsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Restore default settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveSettingsButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save settings as default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="radiantTab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Radiant</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="activeRealData">
         <property name="title">
          <string>Active Radiant - Real Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <widget class="QCheckBox" name="showARR">
               <property name="text">
                <string>Show all markers of active radiant based on real data.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="changeColorARR">
               <property name="text">
                <string>Change Color</string>
               </property>
              </widget>
             </item>
            </layout>
            <zorder>changeColorARR</zorder>
            <zorder>showARR</zorder>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_6">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="textureARR">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>64</width>
               <height>64</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/MeteorShowers/radiantSetting.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="activeGenericData">
         <property name="title">
          <string>Active Radiant - Generic Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_4">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="textureARG">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>64</width>
               <height>64</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/MeteorShowers/radiantSetting.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <widget class="QCheckBox" name="showARG">
               <property name="text">
                <string>Show all markers of active radiant based on generic data.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="changeColorARG">
               <property name="text">
                <string>Change Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="inactive">
         <property name="title">
          <string>Inactive Radiant</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>9</number>
             </property>
             <item>
              <widget class="QCheckBox" name="showIR">
               <property name="text">
                <string>Show all markers of inactive radiant.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="changeColorIR">
               <property name="text">
                <string>Change Color</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="frame_5">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="textureIR">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>64</width>
               <height>64</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/MeteorShowers/radiantSetting.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="labelsGroup">
         <property name="title">
          <string>Label</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Label font size (pixels):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="fontSizeSpinBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string comment="tab in plugin windows">About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextBrowser" name="aboutTextBrowser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="openLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
